Better readability(Personal opinion). (#15380)
mdn/content: files/en-us/web/javascript/reference/functions/default_parameters/index.md

### Evaluated at call time

The default argument is evaluated at _call time_. unlike (for example) Python,

> So, a new object is created each time the function is called at here.

Reviewer: I think this is even better.

```suggestion
So, a new object is created each time the function is called.
```

Reviewer 2: Yes, this is better. But the whole thing needs work. Hold on.

> The default argument is evaluated at _call time_. unlike (for example) Python,
> So, a new object is created each time the function is called at here.

Reviewer 2: Here we go.
```suggestion
The default argument is evaluated at _call time_.
Unlike with Python (for example), a new object is created each time the function is called.
```
